### ISSUE TITLE
Resource download missing header

### ISSUE THREAD

**Issue by makkmarci13** [SpiGetOrg/Spiget#36]
Hello,
 I use spiget for search and download plugins in my mcpanel, but i noticed that the **Content-Disposition** was removed from the download response headers, and this cause that i can't get the filename. Anybody has any idea where can i get the filename? 
Thanks!

**Comment by InventivetalentDev** [comment]
The download endpoint was changed to redirect to the new cdn service. Follow that redirect and it should give you a filename again, although you really shouldn't be relying on that too much.

**Comment by makkmarci13** [comment]
It fallows the redirect, because if i set a static name, it can download it. Is there an api endpoint which provide the jar name? Or how will be better?

**Comment by InventivetalentDev** [comment]
> Is there an api endpoint which provide the jar name  

No. Spiget has no idea what the original file name is. The [previous download route](https://github.com/SpiGetOrg/api.spiget.org/commit/913134d0d2b70825f7a11d73b4e9c46e0aadd625#diff-abfc957f6505bff689b4727079c62705290a444e2a5de1a9b9fa9a757b781b29L190) used to literally set it to `$name#$id#type`, so just use the name from the cdn (which should be the resource id) or set it to whatever you'd like, since it really doesn't matter.
